debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/PostgresConnectorIT.java

```diff
@@ -264,19 +264,6 @@ public void shouldValidateReplicationSlotName() throws Exception {
         assertConfigurationErrors(validatedConfig, PostgresConnectorConfig.SLOT_NAME, 1);
     }
 
-    @Test
```

Comment:
I had to delete the test as there was no way to easily toggle the WAL_LEVEL in the integration test and anything that I'd add would probably introduce more complexity and flakiness than a change like this is worth.